.github/dependabot.yml

```
@@ -0,0 +1,22 @@
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 5
+
+  # 3) Docker images (e.g. Node.js base images in Dockerfile / docker-compose.yml)
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 5
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)